pkg/config/config.go

```diff
@@ -516,6 +516,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("dogstatsd_context_expiry_seconds", 300)
 	config.BindEnvAndSetDefault("dogstatsd_origin_detection", false) // Only supported for socket traffic
 	config.BindEnvAndSetDefault("dogstatsd_origin_detection_client", false)
+	config.BindEnvAndSetDefault("dogstatsd_origin_optout_enabled", true)
```

Comment:
What's your reasoning behind adding a feature flag? Are you afraid of any future misbehaving/misuse?

Reply:
On the odd chance that the new tag value is being used already somewhere, and this feature causes unintended behavior.